java/java.kit/nbproject/project.xml

```diff
@@ -237,6 +237,12 @@
                         <specification-version>2.23</specification-version>
                     </run-dependency>
                 </dependency>
+                <dependency>
+                    <code-name-base>org.netbeans.modules.languages.antlr</code-name-base>
+                    <run-dependency>
+                        <specification-version>1.0</specification-version>
+                    </run-dependency>
+                </dependency>
```

Comment:
Why is language support for antlr4 in the java cluster? antlr grammars can be used to generate lexers/parsers for various languages, java only one of them.

Reply:
Well I planned to support Java actions (inline source elements) only. Right now the LanguageEmbedding is only depending on Java Language Mime-Type, so this module can be put to ide or extide clusters. Though if I'd add Java parsing I'd need more from the Java modules. That's the reason putting it there.

If we would like to be more generic, then this module could again go to ide or extide, and provide some spi-s for the supported languages to implement in their language dependent module. Right now I would be just happy if I can get the Java parsing working.